Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -572,6 +572,7 @@ class SendAnonymousUsageDataSetting: BoolSetting {
             settingDidChange: {
                 AdjustIntegration.setEnabled($0)
                 LeanplumIntegration.sharedInstance.setEnabled($0)
+                LeanplumIntegration.sharedInstance.setUserAttributes(attributes: ["Telemetry Opt Out" : $0])
```

Comment:
Yes, in the future we will be making setEnabled shut off LeanPlum immediately: https://bugzilla.mozilla.org/show_bug.cgi?id=1353820